### PR TITLE
chore(docs): Trim cStor quickstart

### PR DESCRIPTION
**Pull request by niladrih**
Updated cStor quickstart.

Signed-off-by: Niladri Halder <niladri.halder@mayadata.io>

### CODE REVIEW

File: docs/quick.md [hunk 1]
```diff
@@ -137,26 +160,42 @@ follwing steps:
...
-               kubernetes.io/hostname: "worker1"
-             dataRaidGroups:
-             - blockDevices:
-                 - blockDeviceName: "blockdevice-10ad9f484c299597ed1e126d7b857967"
-             poolConfig:
-               dataRaidGroupType: "stripe"
+   apiVersion: cstor.openebs.io/v1
+   kind: CStorPoolCluster
+   metadata:
+     name: cspc-stripe
```
Comment (kmova) [inline docs/quick.md:166]: The word "stripe" in the cluster name might confuse the users to think that the data is stripped across the nodes. Can we just keep this as "cstor-storage"

File: docs/quick.md [hunk 2]
```diff
@@ -2,40 +2,63 @@
...
-| RHEL/CentOS      | iscsi-initiator-utils | <ul><li>sudo yum install iscsi-initiator-utils -y</li><li>sudo systemctl enable --now iscsid</li><li>modprobe iscsi_tcp</li><li>echo iscsi_tcp >/etc/modules-load.d/iscsi-tcp.conf</li></ul> | sudo systemctl status iscsid.service |
-| Ununtu/ Debian   | open-iscsi            |  <ul><li>sudo apt install open-iscsi</li><li>sudo systemctl enable --now iscsid</li><li>modprobe iscsi_tcp</li><li>echo iscsi_tcp >/etc/modules-load.d/iscsi-tcp.conf</li></ui>| sudo systemctl status iscsid.service |
-| RancherOS        | open-iscsi            |  <ul><li>sudo ros s enable open-iscsi</li><li>sudo ros s up open-iscsi</li></ui>| ros service list iscsi |
+| RHEL/CentOS      | iscsi-initiator-utils | <ul><li>sudo yum install iscsi-initiator-utils -y</li><li>sudo systemctl enable --now iscsid</li></ul> | sudo systemctl status iscsid.service |
+| Ubuntu/Debian   | open-iscsi            |  <ul><li>sudo apt install open-iscsi -y</li><li>sudo systemctl enable --now iscsid</li></ui>| sudo systemctl status iscsid.service |
 
+**Note: RancherOS is not a supported platform.**
```
Comment (kmova) [inline docs/quick.md:17]: q_: why is this not supported? Is there an issue that we can point to?

Comment (niladrih) [reply docs/quick.md:17]: Can we use this issue?
https://github.com/rancher/os/issues/3017
cc: @prateekpandey14

Comment (sonasingh46) [reply docs/quick.md:17]: @niladrih -- Can't it be something related to DigitalOcean? Have we tried RancherOS on some lab setups?

Comment (niladrih) [reply docs/quick.md:17]: Yes. But, didn't create an issue. I'll create a cluster, and raise if it is incompatible. Removing the RancherOS not supported message.

**Review by kmova — APPROVED** [review]

**Pull request merged**